cloud/pkg/synccontroller/objectsync.go

```diff
@@ -13,6 +11,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog/v2"
+	"strconv"
```

Comment:
goimport should be executed to format the import modules.

Reply:
sorry, my IDE did it automatically.